index.md

```diff
@@ -89,19 +89,55 @@ as to mitigate subjectivity in role assignment.
-An issue can't be addressed if no one else is aware it exists. Either bring up
-the issue directly with the other member or speak to some of the other members
-that you are close with. If they agree that it's an issue that needs
-addressing, the matter should be brought to the community as a whole for
-discussion.
-
-### [Sanctions](#sanctions)
-* Apologize when appropriate
-* Ameliorate the situation (e.g. repair or replace a damaged game; buy person a
-  lunch to mend things)
-
-Continued violations may result in ejection from the community, by process of
-consensus of active and long-standing members.
+An issue can't be addressed if no one else is aware it exists. Ideally, members
+should bring up the issue directly with the other member.  The next recourse is
+to initiate a formal conflict resolution proceeding.  Before doing so, it is
+highly recommended that a member speak with one or two other members about the
+matter.  If they agree that it's an issue that needs addressing, the matter
+should be brought to the attention of the Adepts and will move through the
+conflict resolution procedure described below.  Note that it is not necessary to
+relay the particulars of a conflict to the Adepts in order to begin conflict
+resolution.
+
+### Conflict Resolution
+
+#### Mediation
+Conflict resolution will initially begin with a mediation phase.  A mediator
+will be selected from the Adept group either by mutual agreement between the
+two conflicting parties or via selection of a disinterested party by the
+Adepts.
+
+The mediator will be responsible for facilitating a discussion between the two
+parties, attempting to ascertain the facts behind the conflict, and for
+proposing a solution amicable to both groups.  Mediation is expected to remain a
+private matter between the two parties and the mediator.  If the two parties are
+still unable to reach an accord, the process moves to arbitration.
+
+#### Arbitration
+Both parties will write a document outlining their version of events and the
+nature of the conflict.  Five disinterested adepts will be selected at random to
+serve as arbiters.  The mediator will present any relevant background and should
+the arbiters have additional questions, the mediator may be dispatched to gather
+additional information.  The arbiters will discuss the matter alone and assign
+sanctions.  Sanctions include:
+
+  * A formal warning
+  * A temporary ban on gaming
+  * A temporary ban from Discord
+  * A permanent ban on gaming
+  * A permanent ban from Discord
+  * A permanent ban from Zargon
+  * Other restorative actions that are deemed appropriate: e.g. replacing a
+    damaged game
+
+A majority of arbiters must agree on any sanction.  The exception to that rule
+is a permanent ban from Zargon.  That decision must be unanimous.
+
+Refusal by either party to accept the arbiters' decision gracefully will result
+in further and more severe sanction.  The mediator will be responsible for
```

Comment:
For my own clarification, the implication here is that begrudging or hostile acceptance of the arbiters' decision would be punished harder also?

Reply:
Yes, but the primary goal is to discourage people from trying to flout or circumvent the decision of the arbiters.